app/src/server/utils/constructLoggedCallFiltersQuery.ts

```diff
@@ -31,8 +29,41 @@ const comparatorToSqlExpression = (comparator: (typeof comparators)[number], val
   };
 };
 
-export const constructFiltersQuery = (
-  filters: z.infer<typeof logFiltersSchema>,
+export const dateComparatorToSqlExpression = (
+  comparator: (typeof comparators)[number],
+  value: number[],
```

Comment:
It loks like this is always supposed to be of length 2? If so you can define it as a tuple eg. `value: [number, number]`.